### PR TITLE
Proxy oa component

### PR DESCRIPTION
**Pull request by LedaLima**
Completed functionality for Proxy OA:
- Read proxy_results.csv
- Convert Proxy response code 
- Add GTI reputation 
- Add Thread Exchange reputation 
- Add empty column uri_sev and network_context
- Create Details file
- Parametrized table name in Engine class

### CODE REVIEW

File: oa/proxy/proxy_oa.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+                update_rows = [[conn[0]] + [conn[1]] + [conn[2]] + [conn[3]] + [conn[4]] + [proxy_iana.get_name(conn[5],"proxy_http_rcode")  if proxy_iana else conn[5]] + [conn[6]] + [conn[7]] + [conn[8]] + [conn[9]] + [conn[10]] + [conn[11]] + [conn[12]] + [conn[13]] + [conn[14]] for conn in rows]
+                update_rows = filter(None, update_rows)
+                header = ["p_date","p_time","clientip","host","webcat","respcode","reqmethod","useragent","resconttype","referer","uriport","serverip","scbytes","csbytes","fulluri"]
+                update_rows.insert(0,header)
+ 
+
+            # create edge file.
+            self._logger.info("Creating edge file:{0}".format(edge_file))
+            with open(edge_file,'wb') as proxy_details_edge:
+                writer = csv.writer(proxy_details_edge, quoting=csv.QUOTE_ALL)
```
Comment (daortizh) [inline oa/proxy/proxy_oa.py]: csv.writer should not be quoting any field. D3's csv parser is unable to handle quotation, any delimiter inside the quotation will end up as a new column on the UI :(

Comment (LedaLima) [inline oa/proxy/proxy_oa.py]: Fixed!

**Comment by natedogs911** [comment]
LGTM

**Comment by EverLoSa** [comment]
LGTM

**Comment by rabarona** [comment]
LGTM

**Pull request merged**